rotkehlchen/exchanges/coinbase.py

```diff
@@ -1038,22 +1089,37 @@ def _deserialize_history_event(self, raw_data: dict[str, Any]) -> HistoryEvent |
                 )
 
             amount_data = raw_data.get('amount', {})
-            amount = deserialize_asset_amount(amount_data['amount'])
+            amount = deserialize_asset_amount_force_positive(amount_data['amount'])
             asset = asset_from_coinbase(amount_data['currency'], time=timestamp)
             notes = raw_data.get('details', {}).get('header', '')
             tx_type = raw_data['type']
+            event_identifier = f'{CB_EVENTS_PREFIX}{raw_data["id"]!s}'
+            timestamp_ms = ts_sec_to_ms(timestamp)
             if notes != '':
                 notes += f' {"from coinbase earn" if tx_type == "send" else "as " + tx_type}'
+
+            if tx_type in ('staking_transfer', 'unstaking_transfer'):
```

Comment:
Shouldn't each one of these types also have their own completely unique `notes`? Except the last one, the final else.

I mean I can imagine what notes should be for all of them. Why just go for an empty or generic one? Unless coinbase always has its own. If it does not though we should have our own.